xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -787,12 +794,13 @@ static void DisplayReconfigured(CGDirectDisplayID display,
 
 NSRect CWinSystemOSX::GetWindowDimensions()
 {
+  NSRect frame;
```

Comment:
is it correct to return uninitialized variable? Maybe better return zero frame - `NSZeroRect`?